geopandas/base.py

```diff
@@ -135,6 +134,31 @@ def area(self):
         ``GeoSeries``."""
         return _delegate_property("area", self)
 
+    @property
+    def crs(self):
+        """
+        :getter: Returns a ``pyproj.CRS`` or None.
+        :setter: Sets the value of the crs.
+            The value can be anything accepted
+            by ``pyproj.CRS.from_user_input()``:
+
+            - CRS WKT string
+            - An authority string [i.e. "epsg:4326"]
+            - An EPSG integer code [i.e. 4326]
```

Comment:
I am not sure we should promote too much all those different formats by listing them here as well. Maybe we can summarize it as "a `pyproj.CRS` or any value that is accepted by `pyproj.CRS.from_user_input()`, such as an authority string (eg "EPSG:4326") or a WKT string." 

That way we also point people to the recommended formats. Also, normally with intersphinx, we should be able to link ```:meth:`pyproj.CRS.from_user_input` ```, I think.

Reply:
Agreed 👍

Reply:
Just updated docs for pyproj to allow intersphinx, will not work until next release of pyproj.